### ISSUE TITLE
[Docs]: Command not updated for production mode of fleet server installation.

### ISSUE THREAD

**Issue by amolnater-qasource** [elastic/beats#30129]
**Description:**
Command not updated for production mode of fleet server installation.
- No fingerprint flag available in the command.
- `--fleet-server-es-ca` flag is changed to `--fleet-server-es-ca-trusted-fingerprint` on 8.0.
- Command format is changed on 8.0.

**Guide Link:**
https://www.elastic.co/guide/en/fleet/8.0/add-a-fleet-server.html
- [Self-Managed]

**Screenshot:**
![7](https://user-images.githubusercontent.com/77374876/151921257-9a021067-d0cc-4188-8c43-62504ba7beeb.JPG)

**Actual Command available on 8.0 rc2 BC-4 self-managed Kibana:**
![6](https://user-images.githubusercontent.com/77374876/151921632-d4350b3e-af05-48fd-a8bd-7f9b90185d94.JPG)

**Comment by amolnater-qasource** [comment]
@manishgupta-qasource Please review.

**Comment by manishgupta-qasource** [comment]
Reviewed & mention to @dedemorton

**Comment by amolnater-qasource** [comment]
FYI @jlind23

**Comment by jlind23** [comment]
Hi @amolnater-qasource @manishgupta-qasource 
The docs PR was created here: https://github.com/elastic/observability-docs/pull/1467/files and it indicates this option. 
Did I miss something?
For your information @michel-laterman worked on this.

**Comment by amolnater-qasource** [comment]
Hi @jlind23 
The docs are still not updated, however the provided PR was merged 13 days ago.
Could you please confirm when we can retest this?

Thanks!

**Comment by dedemorton** [comment]
@jlind23 @michel-laterman There are two command examples on that page:

The first one shows the command to install a Fleet Server when you're using self-signed certs. It includes the fingerprint flag. The second example  shows (I assume) the command to install a Fleet Server when your using certificates signed by a CA. Not sure if the second one was overlooked, or if it's correct for the context.

TBH it's really confusing from the CLI help to figure out which of these options are required in specific contexts. Someone on the dev team needs to go through the following files and make sure the docs are accurate. I've tested them every time I've updated them, but it's been awhile:

https://www.elastic.co/guide/en/fleet/8.0/secure-connections.html
https://www.elastic.co/guide/en/fleet/8.0/elastic-agent-cmd-options.html#elastic-agent-install-command
https://www.elastic.co/guide/en/fleet/8.0/add-a-fleet-server.html
